chore: document that issue comment needs to be in main to work (#2320)
TobikoData/sqlmesh: docs/integrations/github.md

    - created
```

> Note: `issue_comment` event will not work until this change in merged to your main branch. 

Reviewer: ```suggestion
Note: `issue_comment` event will not work until this change is merged into your main branch. 
```

> Therefore to enable this you will need to make the change in a branch, merge, and then future branches will support the deploy command. 

Reviewer: ```suggestion
Therefore, to enable this you will need to make the change in a branch, merge, and then future branches will support the deploy command. 
```
